iris/src/iris/common/tutor_suggestion.py

```
@@ -0,0 +1,258 @@
+    LECTURE = "lecture"
+    GENERAL = "general"
+
+
+def get_user_query(chat_history: List[PyrisMessage]):
+    """
+    Extracts the user query from the chat history.
+    :param chat_history: List of messages in the chat history.
+    :return: The user query as a string.
+    """
+    if chat_history:
+        if chat_history[-1].sender == IrisMessageRole.USER:
```

Ruff (0.12.2)

Use a single `if` statement instead of nested `if` statements

Combine `if` statements using `and`

(SIM102)